generate_manifest.py

```diff
@@ -30,40 +30,53 @@
 import config
 
 
+def required_to_deploy_this_project(project, specific_projects):
+    return len(specific_projects) == 0 or project in specific_projects
+
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('-v', '--versions', default={}, type=json.loads, 
-                        help="Supply specific versions of projects that you like in the manifest. Supplied as a string dictionary. (Will need to escape quotes)")
+                        help="Supply specific versions of projects that you like in the manifest. "
+                             "Supplied as a string dictionary. (Will need to escape quotes)")
     parser.add_argument('-e', '--environment', default="local", type=str,
                         help="Create a manifest based on an existing deploy to an environment.")
+    parser.add_argument('-p', '--projects', default=None, type=str,

```

Comment:
Probably wouldnt do this as a `str` ... look at doing it with `nargs="+"` e.g. https://docs.python.org/3/library/argparse.html#nargs

Reply:
Good idea, will do so